helm-charts/sawmills-remote-operator-manual/tests/proxy_test.yaml

```
@@ -0,0 +1,79 @@
+
+  # 4a. If proxy.http or proxy.https is set and proxy.noProxy is set (single string),
+  #    NO_PROXY should include "<proxy.noProxy>, static entries and $(POD_IP)"
+  - it: should append POD_IP when proxy.https and proxy.noProxy (string) are set
+    set:
+      proxy:
+        https: URL
+        noProxy: " xxx "
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[0].env[?(@.name=="NO_PROXY")].value
+          value: "$(KUBERNETES_SERVICE_HOST),.cluster.local,.svc,.svc.cluster.local,127.0.0.1,::1,kubernetes,kubernetes.default.svc,localhost,xxx,$(POD_IP)"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)